.github/workflows/test.yml

```diff
@@ -8,12 +8,12 @@ on:
     branches: [master]
 
 jobs:
-  check:
+  test:
+    name: Run forge test
+    runs-on: ubuntu-latest
```

Comment:
For consistency, should we also modify the name and order in `deploy.yml`?

Reply:
Updated 🙏